src/style.css

```diff
@@ -0,0 +1,138 @@
+html {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+}
+
+.conteiner {
+  position: relative;
+}
+
+ul {
+  margin-top: 0;
+  margin-bottom: 0;
+  padding: 0;
+}
+
+li {
+  list-style: none;
+  display: flex;
+}
+
+.stars--0 {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.stars--1 {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.stars--2 {
+  display: flex;
+}
+
+.stars--3 {
+  display: flex;
+}
+
+.stars--4 {
+  display: flex;
+}
+
+.stars--5 {
+  display: flex;
+}
+
+.stars__star:not(:first-child) {
+  content: "";
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star {
+  content: "";
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:first-child {
+  content: "";
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  margin-right: 4px;
+}
+
+.stars--2 .stars__star:first-child {
+  content: "";
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  margin-right: 4px;
+}
+
+.stars--2 .stars__star:nth-of-type(2) {
+  content: "";
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  margin-right: 4px;
+}
+
+.stars--3 .stars__star:nth-last-of-type(n+3) {
+  content: "";
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  margin-right: 4px;
+}
+
+.stars--4 .stars__star:nth-of-type(-n+4) {
+  content: "";
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  margin-right: 4px;
+}
+
+.stars--5 .stars__star:nth-of-type(-n+5) {
+  content: "";
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  margin-right: 4px;
+}
```

Comment:
You should give default styles for `.stars__star` and than using nth-child change only background image for each modifier